locales-pending/onboarding.ftl

```diff
@@ -50,6 +50,7 @@ onboarding-enter-details-label-last-name = Last name
 onboarding-enter-details-placeholder-last-name = Enter last name
 onboarding-enter-details-label-location = City and state
 onboarding-enter-details-placeholder-location = Enter city and state
+onboarding-enter-details-placeholder-location-results = No locations found for search
```

Comment:
This sounds strange. Does it mean that no location was found when searching? If that's the case, wouldn't "No location found" be enough?

Reply:
Yes, that’s what it’s supposed to mean. Ran this by content and “No location found” works, thanks!